Formula/python3.rb

```diff
@@ -310,7 +310,22 @@ def post_install
   end
 
   def xy
-    version.to_s.slice(/(3\.\d)/) || "3.6"
```

Comment:
If it's the installed files, you can read it out of any of them in post install.